packages/frontend/core/src/modules/collection-rules/impls/order-by/integration-type.ts

```
@@ -0,0 +1,41 @@
+import type { OrderByParams } from '../../types';
+
+export class IntegrationTypeOrderByProvider
+  extends Service
+  implements OrderByProvider
+{
+  constructor(private readonly docsService: DocsService) {
+    super();
+  }
+
+  orderBy$(
+    _items$: Observable<Set<string>>,
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)